.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: "Visual Tests"
+
+on:
```

Comment:
Maybe we should have this only for changes in the packages? We shouldn't run visula tests if we update the readme or change something in the build

Reply:
Yeah. We should on run this if there are package changes. Let's also rename this file to `on_pull_requests_vrt.yml`.

Reply:
This will run for `on_pull_requests` and also for pushing to main. How would this file name be better for these two actions?

Reply:
I added this to this action: 
```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
  push:
    paths:
      - "packages/**"
    branches:
      - main
```

But how can I add a check in the current branch? I mean, we want to check this for the main and also for the branch that belongs to the PR.

Reply:
Can you do:
```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths:
      - "packages/**"
  push:
    paths:
      - "packages/**"
    branches:
      - main
```

Reply:
I did https://github.com/Localitos/pluto/pull/31/files#diff-164c684a80d38b999b17c7abe8d6485a68155c70969e986bdd1e53386b42eb1eR6 
And it worked. But I'm not sure if this is good. Sometimes the publish to Chromatic might not work and our next push wouldn't be a change in the packages folder, for example, so that it wouldn't trigger the job. 🤷🏻‍♀️ 

I tried this case on my machine and it did trigger the Visual tests job, but I'm still not sure if this will always work as we expect x)

Reply:
Yeah, exactly. I updated the packages and ran this build https://www.chromatic.com/build?appId=632b1baf1257171b6a84f370&number=25
But then I updated it back and committed. But instead of waiting to finish the jobs, I decided to merge with main and then committed again. It didn't generate a new build with the update that I did in the packages. 

I prefer doing it this way:

```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
  push:
    paths:
      - "packages/**"
    branches:
      - main
```

Running in the `packages` folder only for branch `main`. And I'm also not sure if this will work correctly using the same action for two different situations. I'm also open to trying and seeing if it works smoothly. 🤞🏻

Reply:
Yeah. Let's try this method out, and we can always adjust if necessary.